common/f3d_materials.md

```diff
@@ -0,0 +1,193 @@
+
+* Primitive LOD Fraction This register exists simply for free use
+
+* Chroma Key Scale: Made to support chroma key usage. While not using chroma key, simply exists as a free use RGB register.
+
+* YUV Convert K4, YUV Convert K5: Both registers are made to support YUV to RGB conversion. If not converting, these registers are free to use.
+
+::::
+::::{tab-item} Geo
+
+The geo tab refers to geometry mode selection, which affects how triangles are processed while loading or drawing. Some options require other options to be enabled, or change what values certain variables will refer to. These nuances should be described with tooltips when hovering over the options in fast64.
+
+## Z Buffer
+
+Enabling [Z buffer]{.key_text} will make it so that Z buffer calculations are carried out on drawn triangles. This makes it so that triangles can utilize depth checking per pixel if enabled inside the RDP. Disabling Z buffer will change the drawing algorithm of the triangles sent to the RDP to ignore depth. Depth values are never written to triangles in DMEM and when rasterizing depth is never considered for the entire primitive.
+
+## Shading
+
+Enabling [Shade]{.key_text} will make it so shade coefficients are calculated and stored while loading the triangle. These values are used for the `Shade` and `Shade Alpha` input to the combiner. If disabled, coefficients will not be calculated, and instead the previous values left in DMEM will be used.
+
+Disabling shade will also disable fog, as that uses the `Shade Alpha` variable to function.
+
+## Cull Front/Back
+
+Culling will disable the drawing of triangles whose normals face (front) and don't face (back) the camera. Frontface culling is unfortunately not displayed in fast64 as a limitation of blender.
+
+## Fog
+
+The [Fog]{.key_text} geo mode enables the calculation of fog coefficients, which is stored in shade alpha. Fog is blended on top of the combiner output by setting a render mode to an appropriate one with fog.
+
+## Lighting
+
+Enabling [Lighting]{.key_text} will turn on light calculations of shading is enabled. Light colors will be the combination of ambient, plus the affect of directional lights on the surface of triangles.
+
+## Texture UV Generation/Linear
+
+UV generation will create UVs for the triangle based on the triangles normals. Lighting must be enabled to use either of these options.
+
+[Texture UV Generate]{.key_text} will create UV values equal to the component of the screen space normal on an interval of `[0, 1]`. [Texture UV Generate Linear]{.key_text} will take the trig decomposition those normals instead. Consider the following conversions from normals to UV coordinates.
```

Comment:
```suggestion
[Texture UV Generate]{.key_text} will create UV values equal to the component of the screen space normal on an interval of `[0, 1]`. [Texture UV Generate Linear]{.key_text} will take the trig decomposition of those normals instead. Consider the following conversions from normals to UV coordinates.
```

Note this could use more details, the text and examples aren't too clear if the goal was to be detailed